viewer-commons/src/main/java/nl/b3p/mail/Mailer.java

```diff
@@ -96,7 +100,7 @@ public static void sendMail(String fromName, String fromEmail, String email, Str
      * @param filename Give that attachment a naem.
      * @throws Exception if any
      */
-    public static void sendMail(String fromName, String fromEmail, String email, String subject, String mailContent, File attachment, String filename) throws Exception {
+    public static void sendMailWithAttachment(String fromName, String fromEmail, String email, String subject, String mailContent, File attachment, String filename) throws Exception {
```

Comment:
changing the name and changing the API doesn't make sense to me and I konw for a fact that this will break in some projects using `sendMail` with attachments